public/app/features/explore/utils/links.ts

```diff
@@ -121,10 +121,29 @@ export const getFieldLinksForExplore = (options: {
         }
         return linkModel;
       } else {
+        //TODO some sort of global use of transformationData.mappings
```

Comment:
Considering what we've discussed in https://github.com/grafana/grafana/issues/60965 I think we may not need mappings for now. They would be handy if we had a sampler in place, but hence we start without it we assume the user needs to know name of fields so mapping will probably be redundant now.